docs/source/new-docs/getting-started/installing/index.rst

```diff
@@ -4,3 +4,228 @@
+
+Activate the conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will need to run ``mamba activate geoips`` (or ``conda activate geoips``)
+every time you want to run or work on GeoIPS. Eg.
+
+.. code:: bash
+
+    mamba activate geoips
+
+Install system dependencies directly
+------------------------------------
+
+If you are an expert user, you may want to install the dependencies manually. Do not do this if you already installed
+dependencies with Conda.
+
+Required
+^^^^^^^^
+
+* ``git``
+* ``openblas`` (required for scipy pip install)
+* ``make`` (required for pypublicdecompwt)
+* ``python`` >= 3.9
+
+Optional
+^^^^^^^^
+
+* ``gfortran`` (required for plugins including fortran builds)
+* ``gcc`` and ``g++`` (required for plugins including fortran or C builds, and ARM machines)
+* ``pdflatex`` (optional, for building pdf documentation)
+* ``wget`` (required for downloading test data)
+* ``make`` (optional, for building packages on ARM machines)
+* Test data repos can be installed in `$GEOIPS_TESTDATA_DIR`
+
+Debian
+^^^^^^
+
+For example, on Debian machines, you can install the optional and required dependencies like this:
+
+.. code:: bash
+
+    sudo apt-get install git make libopenblas-dev python3
+    sudo apt-get install gfortran gcc g++ gdal-bin libgdal-dev texlive-latex-base wget make
+
+This takes ~1gb of space.
+
+Install GeoIPS
+--------------
+
+We can use ``pip`` to install all GeoIPS Python dependencies, and GeoIPS itself.
+
+First, clone the GeoIPS git repository:
+
+.. code:: bash
+
+    git clone https://github.com/NRLMMD-GeoIPS/geoips.git $GEOIPS_PACKAGES_DIR/geoips
+
+.. code:: bash
+
+    # Ensure geoips python environment enabled before installing geoips
+    pip install "$GEOIPS_PACKAGES_DIR/geoips"
+
+If you want to install GeoIPS with all optional dependencies, you can use:
+
+.. code:: bash
+
+    # Ensure geoips python environment enabled before installing geoips
+    pip install "$GEOIPS_PACKAGES_DIR/geoips[doc,lint,test,debug]"
+
+The optional dependencies are:
+
+- ``doc``: for building the documentation with Sphinx
+  (the documentation is also available online at
+  https://nrlmmd-geoips.github.io/geoips/)
+- ``lint``: for linting the code (useful for developers)
+- ``test``: for running the tests
+- ``debug``: for debugging the code with IPython/jupyter
+
+Development
+-----------
+
+The installation steps for developers are the same as for normal and/or expert users, except for one step.
+
+Most developers use Conda installations, but any dependency management solution is fine.
+
+When installing geoips, please install all the extras and install in **editable** mode so that changes to the code are
+immediately reflected in the installed package. Eg.
+
+.. code:: bash
+
+    # Ensure geoips python environment enabled before installing geoips
+    pip install -e "$GEOIPS_PACKAGES_DIR/geoips[doc,lint,test,debug]"
+
+See the [ADDING FUNCTIONALITY] page for more details on how to contribute to GeoIPS.
```

Comment:
If people are cloning the repo anyway, why not *just* do the editable install and leave it at that?

Reply:
Using `-e` means the system points to the current folder, whereas usually it's copied elsewhere. So `-e` means if you delete the clone, your install will break.... Not sure if that's important, so I can just leave it as `-e` by default for now

Reply:
Changed to `-e` by default